packages/synapse-react-client/src/lib/containers/ConfirmationDialog.tsx

```diff
@@ -1,99 +1,48 @@
-import CloseIcon from '@mui/icons-material/Close'
-import {
-  Box,
-  Button,
-  ButtonProps,
-  Dialog,
-  DialogActions,
-  DialogContent,
-  DialogProps,
-  DialogTitle,
-  IconButton,
-  Stack,
-  SxProps,
-} from '@mui/material'
+import { Button, ButtonProps } from '@mui/material'
 import React from 'react'
-import { HelpPopover } from './HelpPopover'
+import { DialogBase, DialogBaseProps } from './DialogBase'
 
-export type CloseButtonProps = {
-  sx?: SxProps
-  onClick?: () => void
-}
-
-export const CloseButton: React.FC<CloseButtonProps> = ({
-  sx = { color: 'grey.700' },
-  onClick,
-}) => {
-  return (
-    <IconButton sx={sx} onClick={onClick} aria-label="close">
-      <CloseIcon />
-    </IconButton>
-  )
-}
-
-export type ConfirmationDialogProps = {
-  open: boolean
-  title: string
-  content: string | JSX.Element
+export type ConfirmationDialogProps = DialogBaseProps & {
   confirmButtonText?: string
   confirmButtonColor?: ButtonProps['color']
-  className?: string
+  confirmButtonVariant?: ButtonProps['variant']
```

Comment:
Add `confirmButtonVariant` and `hasCancelButton` props for use in `CreateAccessTokenModal`. Otherwise, changes are related to refactoring to create `DialogBase` component, which can be used in `EntityModal` and `OAuthManagement`